resources/assets/less/bem/livestream-page.less

```diff
@@ -0,0 +1,20 @@
+.livestream-page {
+  padding: 20px;
+  @media @desktop {
+    padding-left: 100px;
+    padding-right: 100px;
+  }
+  &__header {
+    color: @grey-5;
+    padding-bottom: 15px;
+  }
+  &__anchor-twitch,

```

Comment:
Every panel is wrapped with it so it is clickable (javascript solution was breaking with overlaying button)